reference/carts.v3.yml

```diff
@@ -1005,7 +1005,7 @@ components:
               description: Given name for a gift certificate line item.
             theme:
               type: string
-              description: The theme of the gift certificate. The following options are available:`Birthday`, `Boy`, `Celebration`, `Christmas`, `General`, and `Girl`.
+              description: 'The theme of the gift certificate. The following options are available:`Birthday`, `Boy`, `Celebration`, `Christmas`, `General`, and `Girl`.'
```

Comment:
Are these the only possible values? If they are, we could document using `enum` (rather than putting them in the description).

Reply:
@bc-andreadao I think they are examples of some of the options but there can be more. Should I still use `enum`?

Reply:
I did that for my Store Logs API, so my opinion is yes. Though maybe up to codeowners to decide